src/operations/command.ts

```diff
@@ -153,6 +161,14 @@ export abstract class CommandOperation<T> extends AbstractOperation<T> {
       command.maxTimeMS = this.options.maxTimeMS;
     }
 
+    if (
+      this.options.rawData != null &&
+      this.hasAspect(Aspect.SUPPORTS_RAW_DATA) &&
+      maxWireVersion(connection) >= MIN_SUPPORTED_RAW_DATA_WIRE_VERSION
+    ) {
+      command.rawData = this.options.rawData;
+    }
+
```

Comment:
One pain point with command operation is that every operation that subclasses it implicitly supports all the options that command operation supports.  We have https://jira.mongodb.org/browse/NODE-7096 planned for tentatively early next year, which seeks alternatives to command operation.  

Nothing in the implementation here makes use of CommandOperation because you've designed this to use aspects.  What do you think of removing this block of code from CommandOperation entirely and instead putting it in a shared util function (probably called immediately after building the command document in Server.command())?

Reply:
I implemented in this manner because of what's in NODE-7096. Using aspects was my approach to filtering out the options if the command didn't support them. The issue I see with moving this code into `Server.command()` is that it just moves the `rawData` option up a level to `OperationOptions`, which still has the same underlying problem.

My original implementation got around this by adding the option directly to each individual operation's options classes, and then adding the code to add the option to the command in each individual operation. (See: https://github.com/mongodb/node-mongodb-native/pull/4581/files). That just felt like I was adding a lot of duplicate code and the approach in this PR seemed much cleaner.